tsconfig.json

```diff
@@ -29,5 +29,10 @@
     "target": "ES2022",
     "allowJs": false,
     "typeRoots": ["@types", "@types-extensions", "node_modules/@types"]
-  }
+  },
+  "exclude": [
+    "application-templates/starter-typescript/@types",
+    "application-templates/starter-typescript/@types-extensions",
+    "application-templates/starter-typescript/jest.test.setup.ts"
```

Comment:
As mentioned in the other comment, it is excluded in the main `tsconfig.json` but perhaps if it belongs to `@types` it should be deleted from here